### PR TITLE
Practical 02

### PR DESCRIPTION
**Pull request by hemilrajpura-simform**
practical 2 from LMS

### CODE REVIEW

File: Practical-2/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.2",
+    "css-loader": "^1.0.0",
+    "style-loader": "^0.23.0",
+    "webpack": "^4.19.1",
+    "webpack-cli": "^3.1.1",
+    "webpack-dev-server": "^3.1.8"
+  },
+  "scripts": {
+    "start": "webpack-dev-server --mode development"
```
Comment (abhie-sheik) [inline Practical-2/package.json]: script to build the project is missing

File: Practical-2/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const path = require("path");
+const webpack = require("webpack");
+
+module.exports = {
+  entry: "./src/index.js",
+  mode: "development",
```
Comment (abhie-sheik) [inline Practical-2/webpack.config.js]: mode can be production here, why is it static?

**Review by abhie-sheik — APPROVED** [review]

File: Practical-2/package.json (hunk 1 above)
Comment (hemilrajpura-simform) [reply Practical-2/package.json]: I have already committed the changes, i think this is the outdated file, can you please check the last commit     3f8c6dc21829e9ac8862b1a7311d1d784f8b2b72

File: Practical-2/webpack.config.js (hunk 2 above)
Comment (hemilrajpura-simform) [reply Practical-2/webpack.config.js]: These files are outdated as it's showing the outdated label, can you please review my latest commit ?

**Pull request merged**